internal/compiler-interface/src/main/java/xsbti/compile/ScalaInstance.java

```diff
@@ -48,7 +55,11 @@ default File libraryJar() {
 	}
 
 	/** Classpath entry that stores the Scala compiler classes. */
-	File compilerJar();
+	File[] compilerJars();
```

Comment:
In what situation would compilerJars contain more than one jar? Shouldn't the scala3doc jar go to otherJars?

Reply:
In the case of Scala 3 it would contain `scala3-compiler`, `scala3-interfaces` and `tasty-core`

Reply:
All jars that are necessary for compilation with the exception of `scala3-library` and `scala-library` which are in `libraryJars`

Reply:
For the sake of exhaustivity, in Scala 3, `compilerJars` would contain:
```
0 = {File@16712} "/home/piquerez/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala3-compiler_3.0.0-M3/3.0.0-M3/scala3-compiler_3.0.0-M3-3.0.0-M3.jar"
1 = {File@16713} "/home/piquerez/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala3-interfaces/3.0.0-M3/scala3-interfaces-3.0.0-M3.jar"
2 = {File@16715} "/home/piquerez/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/tasty-core_3.0.0-M3/3.0.0-M3/tasty-core_3.0.0-M3-3.0.0-M3.jar"
3 = {File@16716} "/home/piquerez/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/modules/scala-asm/7.3.1-scala-1/scala-asm-7.3.1-scala-1.jar"
4 = {File@16717} "/home/piquerez/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-sbt/compiler-interface/1.3.5/compiler-interface-1.3.5.jar"
5 = {File@16718} "/home/piquerez/.cache/coursier/v1/https/repo1.maven.org/maven2/org/jline/jline-reader/3.15.0/jline-reader-3.15.0.jar"
6 = {File@16719} "/home/piquerez/.cache/coursier/v1/https/repo1.maven.org/maven2/org/jline/jline-terminal/3.15.0/jline-terminal-3.15.0.jar"
7 = {File@16720} "/home/piquerez/.cache/coursier/v1/https/repo1.maven.org/maven2/org/jline/jline-terminal-jna/3.15.0/jline-terminal-jna-3.15.0.jar"
8 = {File@16722} "/home/piquerez/.cache/coursier/v1/https/repo1.maven.org/maven2/com/google/protobuf/protobuf-java/3.7.0/protobuf-java-3.7.0.jar"
9 = {File@16723} "/home/piquerez/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-sbt/util-interface/1.3.0/util-interface-1.3.0.jar"
10 = {File@16724} "/home/piquerez/.cache/coursier/v1/https/repo1.maven.org/maven2/net/java/dev/jna/jna/5.3.1/jna-5.3.1.jar"
```